lib/ui/window.dart

```diff
@@ -358,7 +358,7 @@ class SingletonFlutterWindow extends FlutterWindow {
   ///
   ///  * [WidgetsBindingObserver], for a mechanism at the widgets layer to
   ///    observe when this value changes.
-  List<Locale>? get locales => platformDispatcher.locales;
+  List<Locale> get locales => platformDispatcher.locales;
```

Comment:
Does the web_ui version need an update?

Reply:
No, it already returns non-nullable versions.  That's the problem I'm trying to fix: they shouldn't be different.